lsp-methods.el

```diff
@@ -1585,10 +1584,10 @@ https://microsoft.github.io/language-server-protocol/specification#textDocument_
   (lsp--gethash "sortText" c (gethash "label" c "")))
 
 (defun lsp--sort-completions (completions)
-  (sort completions (lambda (c1 c2)
-                      (string-lessp
-                        (lsp--sort-string c1)
-                        (lsp--sort-string c2)))))
+  (sort (seq-into completions 'list) (lambda (c1 c2)
```

Comment:
Is it better to call `seq-into` later (i.e. `(seq-into (sort ...))`).

Not sure about the performance difference though.